src/plugins/data_view_editor/public/components/form_fields/title_docs_popover.tsx

```diff
@@ -96,6 +97,17 @@ export const TitleDocsPopover: React.FC = () => {
                 <EuiCode>filebeat-*,-filebeat-c</EuiCode>
               </p>
             </li>
+            <li>
+              <p>
+                <FormattedMessage
+                  id="indexPatternEditor.titleDocsPopover.useCrossClusterSearchDescription"
+                  defaultMessage="Precede with a cluster name and a colon sign (:) for a cross cluster search."
```

Comment:
@gchaps Should we mention that wildcards (`cluster*:`) also work with cluster names? and one can provide multiple cluster names with (`cluster1,cluster2:`)

Reply:
How about having one bullet point with 3 examples:

> For cross-cluster search, precede with the cluster name followed by a colon(:).
>cluster1:filebeat-*
>cluster1,cluster2:filebeat-*
>cluster*:filebeat-*

Reply:
@kertal @gchaps Thanks! Extended the examples list to showcase what's possible:

<img width="400" alt="Screenshot 2023-05-04 at 09 33 20" src="https://user-images.githubusercontent.com/1415710/236139658-d5b016a9-c70a-4ad1-9900-4cf6ce7bff73.png">

Reply:
looking good, thx!